lightkurve/lightcurvecollection.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division, print_function
```

Comment:
Let's put the collections classes back in lightcurve.py and targetpixelfile.py. They are only ~50 lines long they don't need their own files.

Reply:
Done